packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/main.tsp

```diff
@@ -38,3 +38,115 @@ union ProvisioningState {
   Deleting: "Deleting",
   Accepted: "Accepted",
 }
+
+@resource("operationStatuses")
+@parentResource(TenantLocationResource)
+model TenantOperationStatus is ArmOperationStatus;
+
+@parentResource(SubscriptionLocationResource)
+model SubscriptionOperationStatus is ArmOperationStatus;
+
+@parentResource(ResourceGroupLocationResource)
+model ResourceGroupOperationStatus is ArmOperationStatus;
+
+@armResourceOperations
+interface OperationStatuses {
+  @scenario
+  @scenarioDoc("""
+      Resource GET operation.
+      Expected path: /providers/Azure.ResourceManager.Models.Resources/locations/westus/operationStatuses/00000000-0000-0000-0000-000000000000
+      Expected query parameter: api-version=2023-12-01-preview
+    
+      Expected response body:
+      ```json
+      {
+        "name": "TenantOperation",
+        "startTime": <any date>,
+        "endTime": <any date>,
+        "percentComplete": 2,
+        "status": "Failed",
+        "ManagementError": {
+          "code": "404",
+          "message": "The error message of status test. ",
+          "target": "The target of status test.",
+          "details": [{
+            "code": "404",
+            "message": "The error message of status test. ",
+            "target": "The target of status test.",
+          }],
+          "additionalInfo": [{
+            "type": "additionalType",
+            "info": "additionalInfo"
+          }]
+        }
+      }
+      ```
+    """)
+  getTenantStatus is ArmResourceRead<TenantOperationStatus>;
```

Comment:
I don't understand the cases that you are adding.

For listing available operations samples, I think it's already in the issue description?
https://github.com/Azure/cadl-ranch/issues/688#issue-2462784827

Reply:
Fixed by commit https://github.com/Azure/cadl-ranch/pull/713/commits/15ef204fdaf3d79825610d42684388946c7b51ea

Reply:
The PR title is `listing available operations`, though the actual case you are adding is `getting operation status`?

Reply:
Fixed by commit https://github.com/Azure/cadl-ranch/pull/713/commits/4e639a8c2464febccec886fcfb3b531b02c0cfb9